src/t2tSupport.cpp

```diff
@@ -0,0 +1,105 @@
+#include <bitcoinapi/types.h>
+#include <iostream>
+#include "libtxref/txref.h"
+#include "txid2txref.h"
+#include "bitcoinRPCFacade.h"
+#include "t2tSupport.h"
+
+namespace t2t {
+
+    void encodeTxid(const BitcoinRPCFacade &btc, const Config &config, struct Transaction &transaction) {
+
+        blockchaininfo_t blockChainInfo = btc.getblockchaininfo();
+
+        // determine what network we are on
+        bool isTestnet = blockChainInfo.chain == "test";
+
+        // use txid to call getrawtransaction to find the blockhash
+        getrawtransaction_t rawTransaction = btc.getrawtransaction(config.query, 1);
+        std::string blockHash = rawTransaction.blockhash;
+
+        // use blockhash to call getblock to find the block height
+        blockinfo_t blockInfo = btc.getblock(blockHash);
+        int blockHeight = blockInfo.height;
+
+        // warn if #confirmations are too low
+        int numConfirmations = blockInfo.confirmations;
+        if (numConfirmations < 6) {
```

Comment:
This constant should make its way to config files, eventually.

Reply:
yes, I have a few magic numbers in the code that I need to pull out.